app/controllers/sessions_controller.rb

```diff
@@ -0,0 +1,37 @@
+class SessionsController < ApplicationController
+  public_permissions only: %i(new create)
+
+  attr_reader :session_create
+  helper_method :session_create
+
+  # Sign in form (start)
+  def new
+    @session_create = Session::Create.new
+  end
+
+  # Sign in (complete)
+  def create
+    @session_create = Session::Create.new(**session_create_params)
```

Comment:
What is the double star here do?

Reply:
It's possible they aren't actually required in this context, but like a `*arr` turns into multiple positional arguments, a `**hash` turns into multiple keyword arguments (as long as the hash keys are symbols, hence the `symbolize_keys` call in that `_params` method).

The `*arr` operator is called a "splat" and the `**hash` is called a double splat.

More reading: https://alexcastano.com/everything-about-ruby-splats/